tests/test_causal_model.py

```diff
@@ -117,72 +117,72 @@ def test_graph_input2(self, beta, num_instruments, num_samples, num_treatments):
         )
         # removing two common causes
         gml_str = """graph[
-        directed 1 
-        node[ id "{0}" 
+        directed 1
```

Comment:
Were these whitespace edits intentional?

Reply:
I believe this is causing the format check to fail in the build process.

Reply:
No, they weren't intentional. Will look into fixing this, might have been caused by some editor auto-formatting.